react/components/EXPERIMENTAL_FilterBar/index.js

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable camelcase */
```

Comment:
⛏ Naming variables without `_` is better than disabling this rule

Reply:
It's a pattern we chose for experimental components :(

Reply:
Ohh, I thought you used this rule because of the `_st` variables :P